src/test/java/com/artipie/front/api/UsersTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2022 artipie.com
+ * https://github.com/artipie/front/LICENSE.txt
+ */
+package com.artipie.front.api;
+
+import com.amihaiemil.eoyaml.Yaml;
+import java.util.Optional;
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.skyscreamer.jsonassert.JSONAssert;
+import spark.Request;
+import spark.Response;
+
+/**
+ * Test for {@link Users}.
+ * @since 0.1
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+class UsersTest {
+
+    @Test
+    void writesUsers() throws JSONException {
+        JSONAssert.assertEquals(
+            new Users(
+                Yaml.createYamlMappingBuilder().add(
```

Comment:
You can reuse classes from https://github.com/artipie/front/blob/master/src/test/java/com/artipie/front/settings/YamlCredentialsTest.java to build credentials (or maybe extract it to some test package)